tee-worker/litentry/core/assertion-build/src/dynamic/contracts/token_holding_amount/TokenHoldingAmount.sol

```diff
@@ -50,17 +49,10 @@ abstract contract TokenHoldingAmount is DynamicAssertion {
 
 		string memory tokenLowercaseName = abi.decode(params, (string));
 
-		if (
-			keccak256(abi.encodePacked(tokenNames[tokenLowercaseName])) ==
-			keccak256(abi.encodePacked(""))
-		) {
-			revert("Token not supported or not found");
-		}
-
```

Comment:
Instead of removing it, better use tokenRanges to check token as below instead?
if (tokenRanges[tokenName].length > 0) {
  revert("Token not supported or not found");
}